backend/apps/mentorship/models/task_level.py

```diff
@@ -0,0 +1,57 @@
+"""TaskLevel model for the Mentorship app."""
+
+from __future__ import annotations
+
+from django.db import models
+
+
+class TaskLevel(models.Model):
+    """Task level model representing difficulty and prerequisites for mentorship tasks."""
+
+    class Meta:
+        db_table = "mentorship_task_levels"
+        verbose_name_plural = "Task Levels"
+        ordering = ["name"]
+
+    description = models.TextField(
+        verbose_name="Description",
+        blank=True,
+        default="",
+    )
+
+    github_labels = models.JSONField(
```

Comment:
Keep the naming consistent -- either `github_labels` or `labels` everywhere (see the `Module` model)